README.md

```diff
@@ -206,18 +208,27 @@ certain test should run, for example, with
 ```
 
 ## Translator usage
+Translators for Puppet, don't use this workflow.  In the Puppet workflow POs are generated in our translation tool, from an up to date POT.  We don't, as developers, update or commit POs. So this may only be relevant should a developer want to test or generate a test language.
 
-When a translator gets ready to translate messages, they need to update the
-corresponding `.po` file. For example, to update German translations,
-they'd run
+### Generate a test .po file
+Prior to generating a po file, make sure the POT is up to date by running `make i18n`.
+This will put new msgids from the app, into the POT.
 
-    make locales/de.po
+To create a `.po` file for the language eo:
 
-and then edit `locales/de.po`. The plugin actually performs the `make`
-invocation above every time you compile the project, so you should only
-have to do it manually to add a PO file for a new locale. Translators
-should be able to work off the PO files that are checked into source
-control, as they are always kept 'fresh' by the plugin.
+    make locales/eo.po
+
+Note this will just take the contents of the current POT and write the PO from it.
+Subsequent runs will not keep that file up to date.
+
+### Update a test .po file
+Prior to updating a po file, make sure the POT is up to date by running `make i18n`.
+This will put new msgids from the app, into the POT.
+
+To update the po:
+    msgmerge -U locales/eo.po locales/<project-name>.pot
+
+This uses the contents of the current POT to update msgids in the target po (eo.po).
```

Comment:
Hmm, when we talked last week, you said that we don't want people to update the PO files, since they should be managed by transifex. If there is cause to update the PO files outside of transifex we should add that functionality back into the Makefile rather than having people call msgmerge themselves.